jobs/e2e-test-cypress.groovy

```diff
@@ -0,0 +1,138 @@
+// Single job for cypress e2e tests.
+// 
+// cypress e2e tests are the smoketests run in the webapp/feature/cypress repo, 
+// that hit a live website using lambdatest cli.
+//
+// TODO(ruslan): integrate later this job to the e2e tests pipeline.
+// This job runs lambdatest-cypress-cli in the webapp environment 
+// depending on how parameters are specified.
+// 
+
+
+@Library("kautils")
+// Classes we use, under jenkins-jobs/src/.
+import org.khanacademy.Setup;
+// Vars we use, under jenkins-jobs/vars/.  This is just for documentation.
+//import vars.exec
+//import vars.notify
+//import vars.withTimeout
+//import vars.withSecrets
+
+public class TestFailed extends Exception {}
+
+new Setup(steps
+
+).addStringParam(
+   "GIT_REVISION",
+   """The name of a webapp branch to use when building.""",
+   "master"
+
+).addStringParam(
+   "URL",
+   "The url-base to run these tests against.",
+   "https://www.khanacademy.org/"
+
+).addStringParam(
+   "SLACK_CHANNEL",
+   "The slack channel to which to send failure alerts.",
+   "#cypress-testing"
+
+).addStringParam(
+   "DEPLOYER_USERNAME",
+   """Who asked to run this job, used to ping on slack.
+Typically not set manually, but rather by other jobs that call this one.""",
+   "username"
+
+).addStringParam(
+   "REVISION_DESCRIPTION",
+   """Set by the buildmaster to give a more human-readable description
+of the GIT_REVISION, especially if it is a commit rather than a branch.
+Defaults to GIT_REVISION.""",
+   ""
+
+).addStringParam(
+   "NUM_WORKER_MACHINES",
+   """How many worker machines to use. Max available is 20.""",
+   "20"
+
+).addStringParam(
+   "TEST_RETRIES",
+   """How many retry attempts to use. By default is 1.""",
+   "1"
+
+).addChoiceParam(
+   "TEST_TYPE",
+   """\
+<ul>
+  <li> <b>all</b>: run all tests</li>
+  <li> <b>deploy</b>: run only those tests that are important to run at
+        deploy-time (as identified by the `@`
+        decorator)</li>
+  <li> <b>nightly</b>: run only night list of tests.</li>
+</ul>
+""",
+   ["all", "deploy", "nightly"]
+
+
+).apply()
+
+def _setupWebapp() {
+   GIT_SHA1 = kaGit.resolveCommitish("git@github.com:Khan/webapp",
```

Comment:
This isn't a global so you can just use `def`.  Though I think you can probably just call:
```
kaGit.safeSyncToOrigin("git@github.com:Khan/webapp", GIT_REVISION);
```
?  You don't need to store the sha1 because you're not needing to use the same sha1 again elsewhere, like with the job you copied from.